components/director/internal/domain/application/service.go

```diff
@@ -250,18 +258,18 @@ func (s *service) Create(ctx context.Context, in model.ApplicationRegisterInput)
 
 	err = s.labelUpsertService.UpsertMultipleLabels(ctx, appTenant, model.ApplicationLabelableObject, id, in.Labels)
```

Comment:
you could log that we are updating labels and dump the new labels

Reply:
Here when a new Label is created (because we are in creation flow in that function) its `id` is logged along with the respective object type and its `id`. Do you mean to log the key-value pair of each label as well? If so, should it be on `Info` or `Debug` level?

Reply:
I think labels can be on debug